packages/transactions/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -42,6 +42,7 @@ const ListViewToolBar: FC<{
 
   const { selectedRows } = useTableStore(state => ({
     selectedRows: Object.keys(state.rowState)
+      .filter(id => state.rowState[id]?.isSelected)
```

Comment:
somewhat offroad pagination fix: If you use `cmd+ +/-` the number of rows increases, but when you reduce the number of rows rendered, the pagination "showing x-y" doesn't get updated as it's using the total number of rows in memory rather than number of rows being rendered

Reply:
I'm not seeing the pagination bit change when I reduce the number of rows. should I?

Reply:
I've forgotten what I was talking about 😁 

I'll check on the pagination after doing some mergin 🤣